### PR TITLE
refactor: update references from ubiquibot to ubiquity-os

### PR DESCRIPTION
**Pull request by gentlementlegen**
Replaced all instances of 'ubiquibot' with 'ubiquity-os' in comments and test data.

### CODE REVIEW

File: tests/__mocks__/payloads/comment-created.json [hunk 1]
```diff
@@ -89,7 +89,7 @@
     "created_at": "2024-05-19T11:54:23Z",
     "updated_at": "2024-05-19T11:54:23Z",
     "author_association": "CONTRIBUTOR",
-    "body": "/query @ubiquibot",
+    "body": "/query @ubiquity-os",
```
Comment (0x4007) [inline tests/__mocks__/payloads/comment-created.json:92]:
```suggestion
    "body": "/query @UbiquityOS",
```

File: tests/http/request.http [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # org_name: the organization you want to post to, e.g. "ubiquibot"
 # repo_name: the target repository, e.g "command-query-user"
 # issue_number: the number of the issue, same as the one you would see in the url to that issue, e.g. "1"
-# command: the command you want to test, e.g. "/query @ubiquibot"
+# command: the command you want to test, e.g. "/query @ubiquity-os"
```
Comment (0x4007) [inline tests/http/request.http:5]:
```suggestion
# command: the command you want to test, e.g. "/query @UbiquityOS"
```

File: src/handlers/command-parser.ts [hunk 3]
```diff
@@ -11,7 +11,7 @@ export class CommandParser {
     program
       .command("/query")
       .usage("@<username>")
-      .argument("<username>", "User name to query, e.g. @ubiquibot", this._parseUser)
+      .argument("<username>", "User name to query, e.g. @ubiquity-os", this._parseUser)
```
Comment (0x4007) [inline src/handlers/command-parser.ts:14]:
```suggestion
      .argument("<username>", "User name to query, e.g. @UbiquityOS", this._parseUser)
```

**Review by 0x4007 — APPROVED** [review]

**Comment by gentlementlegen** [comment]
@0x4007 Be careful that you are not doing Conventional Commits and you will break the CI.

**Comment by 0x4007** [comment]
> @0x4007 Be careful that you are not doing Conventional Commits and you will break the CI.

This message format should be added to our exceptions. It's from the GitHub UI.

**Pull request merged**